src/main/java/com/iota/iri/network/pipeline/TransactionProcessingPipelineImpl.java

```diff
@@ -69,6 +76,10 @@
     private BlockingQueue<ProcessingContext> broadcastStageQueue = new ArrayBlockingQueue<>(100);
     private BlockingQueue<ProcessingContext> replyStageQueue = new ArrayBlockingQueue<>(100);
 
+    private int getNumberOfThreads() {
```

Comment:
One last thing, usually when a method doesn't use the fields of the class I like to attach `static` to it
I used to think that the JIT optimizes it better but I don't think that's really true

However, I see advantages to it because:
1. it can be used by public static methods
2. when you review/maintain code you know immediately that this method can be reused/moved w/o fear
3. when you change code you know immediately that the output of the method won't be affected due to a change to the state of the class

Reply:
Changed this to a static final int now.